public/components/ImageDrilldown.vue

```diff
@@ -6,18 +6,38 @@
     :title="visibleTitle"
     :visible="visible"
   >
-    <image-label
-      v-if="item && dataFields"
-      class="image-label"
-      :dataFields="dataFields"
-      includedActive
-      :item="item"
-    />
-    <div
-      class="image-container"
-      ref="imageContainer"
-      :style="{ '--IMAGE_MAX_SIZE': IMAGE_MAX_SIZE + 'px' }"
-    ></div>
+    <div class="drill-down">
+      <image-label
+        v-if="item && dataFields"
+        class="image-label"
+        :dataFields="dataFields"
+        includedActive
+        :item="item"
+      />
+      <div
+        class="image-container"
+        ref="imageContainer"
+        :style="{ '--IMAGE_MAX_SIZE': IMAGE_MAX_SIZE + 'px' }"
+      ></div>
+      <div class="slider-container">
+        <label class="slider-label">0.0</label>
+        <input
```

Comment:
Maybe consider using the bootstrap-vue b-form input slider for (as seen in DataSize.vue) for style consistency but otherwise fine.

Reply:
I will add the change to use bootstrap slider.